tests/fast-integration/skr-aws-upgrade-integration/index.js

```diff
@@ -44,6 +44,11 @@ describe('SKR-Upgrade-test', function() {
     return options;
   };
 
+  it(`Perform kcp login`, async function() {
+    console.log('Performing kcp login');
+    await kcp.login();
```

Comment:
how it will help? kcp.login() is called in the getOrProvisionSKR function

Reply:
You are right.